custom-cookbooks/redis/cookbooks/custom-redis/attributes/overrides.rb

```diff
@@ -0,0 +1,2 @@
+# FB-851
```

Comment:
Ditto here. It's already updated in `custom-cookbooks/redis/cookbooks/custom-redis/attributes/default.rb`

Reply:
Yep true, I'll be removing the `overrides.rb` via the next release.